jsdoc.json

```diff
@@ -0,0 +1,79 @@
+{
+	"plugins": [ "jsdoc" ],
+	"settings": {
+		"jsdoc": {
+			"tagNamePreference": {
+				"alternateClassName": "alias",
```

Comment:
Is alternateClassName needed for jsduck compat? Otherwise we should use alias.

Reply:
cc @Krinkle @jdforrester

Reply:
Other than  jqXHR it seems rarely used; let's just ignore it: https://codesearch.wmflabs.org/search/?q=alternateClassName&i=nope&files=&repos=